ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -377,6 +377,42 @@ tests:
         NUM_MASTERS=1
         NUM_WORKERS=0
     workflow: assisted-baremetal
+- always_run: false
+  as: e2e-metal-assisted-osc-4-17
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    dependencies:
+      OPENSHIFT_TESTS_IMAGE: tests-4.17
+    env:
+      ASSISTED_CONFIG: |
+        OLM_OPERATORS=osc
+        NUM_MASTERS=3
+        NUM_WORKERS=3
+        MASTER_MEMORY=10240
+        MASTER_CPU=6
+        WORKER_MEMORY=10240
+        WORKER_CPU=6
+        OPENSHIFT_VERSION=4.17
+        OPENSHIFT_INSTALL_RELEASE_IMAGE=""
+    workflow: assisted-baremetal
+- always_run: false
+  as: e2e-metal-assisted-osc-sno-osc-4-17
+  optional: true
+  steps:
+    cluster_profile: packet-assisted
+    dependencies:
+      OPENSHIFT_TESTS_IMAGE: tests-4.17
+    env:
+      ASSISTED_CONFIG: |
+        OLM_OPERATORS=osc
+        NUM_MASTERS=1
+        NUM_WORKERS=0
```

Comment:
why 1 master?

Reply:
so can you add `sno` to the name of the test ? `e2e-metal-assisted-osc-sno-4-17`

Reply:
@eifrach Updated done. Thanks.